easybuild/easyconfigs/l/libcerf/libcerf-1.11-GCCcore-7.3.0.eb

```diff
@@ -14,12 +14,9 @@ description = """
 toolchain = {'name': 'GCCcore', 'version': '7.3.0'}
 toolchainopts = {'pic': True}
 
-source_urls = [
-    'http://apps.jcns.fz-juelich.de/src/libcerf/',
-    'http://apps.jcns.fz-juelich.de/src/libcerf/old',
-]
-sources = [SOURCE_TGZ]
-checksums = ['70101cac4a0d7863322d4d06cf95c507a9cfd64fc99ad1b31a8425204cfd9672']
+source_urls = ['https://jugit.fz-juelich.de/mlz/libcerf/-/archive/v1.11/']
```

Comment:
@smoors Please use `%(version)s` so this won't need to be touched if `--try-software-version` is used for future releases.